tests/gds_kernel_latency.c

```diff
@@ -193,8 +193,22 @@ static int pp_connect_ctx(struct pingpong_context *ctx, int port, int my_psn,
 
 	ctx->ah = ibv_create_ah(ctx->pd, &ah_attr);
 	if (!ctx->ah) {
-		fprintf(stderr, "Failed to create AH\n");
-		return 1;
+		union ibv_gid dgid;
```

Comment:
you could add a comment describing this fallback code.
e.g.. "this code is required for RoCE V1 and/or V2 blabla"